src/test/java/com/google/api/codegen/testsrc/common/library.proto

```diff
@@ -366,8 +366,7 @@ message Book {
   // Book names have the form `bookShelves/{shelf_id}/books/{book_id}`.
   // Message field comment may include special characters: <>&"`'@.
   string name = 1 [
-    (google.api.field_behavior) = REQUIRED,
-    (google.api.resource_reference).type = "library.googleapis.com/Book"
```

Comment:
Without the source files change, this proto change would result in C# baseline not having a partial class for the resource name in the Book class.